kroxylicious-operator/src/main/java/io/kroxylicious/kubernetes/operator/ProxyDeployment.java

```diff
@@ -77,10 +77,12 @@ static Map<String, String> podLabels() {
     }
 
     private PodTemplateSpec podTemplate(KafkaProxy primary) {
+        PodTemplateSpec podTemplate = primary.getSpec().getPodTemplate();
```

Comment:
From the call earlier I was expecting to see something here which explicitly rejected things from the user supplied pod template that we don't support.

Reply:
Yeah, but the pod template spec supports more than just labels right? And we don't want to pass anything other than labels at this point

Reply:
sure, I'll swap it around so we specifically include the labels rather than using the supplied PodTemplateSpec as the base.